test/Microsoft.ML.Tests/Scenarios/Api/Estimators/TrainWithInitialPredictor.cs

```diff
@@ -22,7 +22,7 @@ public void New_TrainWithInitialPredictor()
 
             var ml = new MLContext(seed: 1, conc: 1);
 
-            var data = ml.Data.CreateTextReader(TestDatasets.Sentiment.GetLoaderColumns(), hasHeader: true).Read(GetDataPath(TestDatasets.Sentiment.trainFilename));
```

Comment:
>GetLoaderColumns [](start = 71, length = 16)

do we still use `GetLoaderColumns()` anywhere? Maybe we can delete the method?

Reply:
we do use it in a few tests. 

---
In reply to: [243121856](https://github.com/dotnet/machinelearning/pull/1878#discussion_r243121856) [](ancestors = 243121856)